docs/source/getting_started/index.rst

```diff
@@ -97,6 +97,23 @@ Optional package dependencies can be installed for specific usage:
 - - `Scooby <https://pypi.org/project/scooby//>`_ for dependency reports
 
 
+Internal versioning
+~~~~~~~~~~~~~~~~~~~
+Please note that the versioning of the various components perfectly matches the following mapping:
+
+------------------------------------
+ansys-dpf-core | 0.2.1   | 0.3.0   |
```

Comment:
@anslpa maybe we could mention that it's all 0.3. ... because we should be creating a new package with the new operator.eval() method and hopefully many others next